internal/venice-client-common/src/main/java/com/linkedin/venice/httpclient5/HttpClient5Utils.java

```diff
@@ -111,6 +112,7 @@ public CloseableHttpAsyncClient build() {
         return HttpAsyncClients.custom()
             .setVersionPolicy(HttpVersionPolicy.FORCE_HTTP_1)
             .setIOReactorConfig(ioReactorConfig)
+            .setConnectionReuseStrategy(new DefaultConnectionReuseStrategy())
```

Comment:
We may want to use `setKeepAliveStrategy` as well to setup keep-alive, otherwise the connection can be closed quickly when idle.

Reply:
If we dont specify it would use `DefaultConnectionKeepAliveStrategy` which had default value of 2 or 3 mins of idle time.

Reply:
In practice, we normally would like to keep the connection alive for at least several hours, not mins when idle.

Reply:
So you want to bump the default to few 1hrs ?

Reply:
Yeah, at least that is the behavior of httpasyncclient.

Reply:
No its not enough I verified that, the main issue as I said is `However when the connection is put back into the pool the state is set to the X500Principal object of the SSL peer.`.  we need to reset the token

Reply:
For the same SSL peer, the `X500Principal` should be same, right?
Or will each request generate a new `X500Principal` every time? Doesn't sounds reasonable.